package/package.json

```diff
@@ -1,16 +1,42 @@
 {
-  "name": "reactmonitorfiber",
-  "version": "1.0.11",
-  "description": "Visualize React's component tree performance and optimization suggestions",
-  "main": "createTree.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/oslabs-beta/ReactMonitor"
+  "_from": "reactmonitorfiber",
+  "_id": "reactmonitorfiber@1.0.11",
+  "_inBundle": false,
+  "_integrity": "sha512-17ru2xLe/osiMB5Dfk2kBCX/1XMPjJ/YBb8B7eBzKlGhe7pevJ82CQgJwHgl/HsKWFJcbuLtiEibQdKTY4pGTQ==",
+  "_location": "/reactmonitorfiber",
+  "_phantomChildren": {},
+  "_requested": {
+    "type": "tag",
+    "registry": true,
+    "raw": "reactmonitorfiber",
+    "name": "reactmonitorfiber",
+    "escapedName": "reactmonitorfiber",
+    "rawSpec": "",
+    "saveSpec": null,
+    "fetchSpec": "latest"
   },
+  "_requiredBy": [
+    "#DEV:/",
+    "#USER"
+  ],
+  "_resolved": "https://registry.npmjs.org/reactmonitorfiber/-/reactmonitorfiber-1.0.11.tgz",
+  "_shasum": "ef17b17b90312e8ae1a006307905317118d88f9f",
+  "_spec": "reactmonitorfiber",
+  "_where": "/Users/usr1/edp/codesmith/react_p2",
   "author": "",
```

Comment:
why is this being added to package.json?

Reply:
maybe this is a result that the checksum of the package directory changed as a result of the modifications; that is, npm uses the checksum for consistency and security... so any modification would cause a modification in package.json of the npm package